btcjson/chainsvrcmds.go

```diff
@@ -895,7 +895,7 @@ func NewSendRawTransactionCmd(hexTx string, allowHighFees *bool) *SendRawTransac
 // sendrawtransaction JSON-RPC command to a bitcoind node.
 //
 // A 0 maxFeeRate indicates that a maximum fee rate won't be enforced.
-func NewBitcoindSendRawTransactionCmd(hexTx string, maxFeeRate int32) *SendRawTransactionCmd {
+func NewBitcoindSendRawTransactionCmd(hexTx string, maxFeeRate float64) *SendRawTransactionCmd {
```

Comment:
Should we add a note in the Godoc comment above what the unit for the `maxFeeRate` is?

Reply:
I added a comment about `maxFeeRate` unit.
https://github.com/btcsuite/btcd/pull/2142/commits/714e994d72681988a1866e79abe136e1aebbb05e  
I will rebase after review.

> We have some of this in lnd

Would it be possible to tell me where in lnd the above mentioned file is located?

I assume that adding a type alias for BTC/kvB will be handled in a separate PR once it is decided, as it requires an update to btcutil.

Reply:
The fee rate types are here: https://github.com/lightningnetwork/lnd/blob/60bc30dd08bb6abb39d7f8481ebd990d587e5469/lnwallet/chainfee/rates.go.

Yes, because of the Go modules, there first needs to be a PR in `btcutil`, then after merging that a tag is pushed and then the code can be used in this file here.

But maybe, because `bitcoind` uses its own fee rate type (`BTC/kvB` instead of `sat/kWU` we use in `lnd`), it would be okay to define the type in this package directly, as AFAIK it's the only place where we actually use `BTC/kvB`.
It looks like the fee rate is used for `fundrawtransaction` and `sendrawtransaction` currently.

Reply:
Thank you for sharing your location.

Yes, I agree with adding the type alias here.
It is used by `TestMempoolAccept`, `fundrawtransaction` and `SendRawTransaction`, so I am replacing their definitions with this type alias

https://github.com/btcsuite/btcd/pull/2142/commits/bd5af2b56fc36c1be1729e459b95d7f575e0134a